public/docs/ts/latest/guide/appmodule.jade

```diff
@@ -0,0 +1,136 @@
+include ../_util-fns
+
+:marked
+  An Angular Module class describes how the application parts fit together.
+  Every application has at least one Angular Module at the _root_ level of the app.
+  You can call it anything you want. The conventional name is `AppModule`.

```

Comment:
I got push back from high places that _Angular doesn't have this convention, we do_. So I'm trying to be more explicit about that.